packages/web/stories/buttons/button/Button.stories.ts

```diff
@@ -16,90 +16,97 @@ ButtonTest.args = {
+    template: `<freud-button label="Button primary"></freud-button>`,
   };
 };
 
 export const Secondary = () => {
   return {
-    template: `<freud-button [color]="'secondary'" [label]="'Button secondary'"></freud-button>`,
+    template: `<freud-button color="secondary" label="Button secondary"></freud-button>`,
   };
 };
 
 export const Ghost = () => {
   return {
-    template: `<freud-button [color]="'ghost'" [label]="'Button ghost'"></freud-button>`,
+    template: `<freud-button color="ghost" label="Button ghost"></freud-button>`,
   };
 };
 
 // Background
 export const BGColorPrimary = () => {
   return {
-    template: `<freud-button [bgColor]="true" [label]="'ButtonBG primary'"></freud-button>`,
+    template: `<freud-button [bgColor]="true" label="ButtonBG primary"></freud-button>`,
   };
 };
 
 export const BGColorSecondary = () => {
   return {
-    template: `<freud-button [bgColor]="true" [color]="'secondary'" [label]="'ButtonBG secondary'"></freud-button>`,
+    template: `<freud-button [bgColor]="true" color="secondary" label="ButtonBG secondary"></freud-button>`,
   };
 };
 
 export const BGColorGhost = () => {
   return {
-    template: `<freud-button [bgColor]="true" [color]="'ghost'" [label]="'ButtonBG ghost'"></freud-button>`,
+    template: `<freud-button [bgColor]="true" color="ghost" label="ButtonBG ghost"></freud-button>`,
   };
 };
 
 // Disabled
 export const Disabled = () => {
   return {
-    template: `<freud-button [disabled]="true" [label]="'Button disabled'"></freud-button>`,
+    template: `<freud-button [disabled]="true" label="Button disabled"></freud-button>`,
   };
 };
 
 // Loading
 export const Loading = () => {
   return {
-    template: `<freud-button [loading]="true" [bgColor]="false" [label]="'Button loading'" [iconPos]="'right'"></freud-button>`,
+    template: `<freud-button [loading]="true" [bgColor]="false" label="Button loading" iconPos="right"></freud-button>`,
   };
 };
 
 // Sizes
 export const SizeLg = () => {
   return {
-    template: `<freud-button [bgColor]="bgColor" [size]="'lg'" [label]="'Button large'"></freud-button>`,
+    template: `<freud-button [bgColor]="bgColor" size="lg" label="Button large"></freud-button>`,
   };
 };
 
 export const SizeMd = () => {
   return {
-    template: `<freud-button [bgColor]="bgColor" [size]="'md'" [label]="'Button medium'"></freud-button>`,
+    template: `<freud-button [bgColor]="bgColor" size="md" label="Button medium"></freud-button>`,
   };
 };
 
 export const SizeSm = () => {
   return {
-    template: `<freud-button [bgColor]="bgColor" [size]="'sm'" [label]="'Button small'"></freud-button>`,
+    template: `<freud-button [bgColor]="bgColor" size="sm" label="Button small"></freud-button>`,
+  };
+};
+
+// Full Width
+export const fullWidth = () => {
+  return {
+    template: `<freud-button [bgColor]="bgColor" [fullWidth]="true" label="Button fullWidth"></freud-button>`,
```

Comment:
```suggestion
    template: `<freud-button [bgColor]="bgColor" fullWidth label="Button fullWidth"></freud-button>`,
```

Poderia ser assim também.

Reply:
Eu testei dessa forma mas não funcionou

Reply:
Umm, era só `fullWidth` mesmo, mas provável que precise alterar algo no componente pra funcionar assim, e acho que não precisamos disso pra agora. Deixa assim mesmo.